### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 6

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#167] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-30 171411](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/a96e9a44-ce7b-4987-9eda-dc6ebfd19bf6)

**Comment by Elia-Belli** [comment, marked as answer]
Lo spazio degli eventi $\Omega=\\{(ABX_1,ACX_2,BCX_3)|X_i=0,1\\}$ dove $X_i$ indica l'agibilità del sentiero (agibile se $1$) e sappiamo che $\mathbb{P}(AB1)=p_{AB}\ ;\ \mathbb{P}(AC1)=p_{AC}\ ;\ \mathbb{P}(BC1)=p_{BC}$.
- **(1)** Dal bivacco $A$ posso recarmi in $C$ se il sentiero $AC$ è percorribile o se lo sono entrambi i sentieri $AB$ e $BC$:
$\mathbb{P}(A\rightarrow C)=\mathbb{P}((AC1)\cup (AB1,BC1))=\mathbb{P}((AC1)\cup ((A\rightarrow B) \cap (B\rightarrow C)))=$
$=\mathbb{P}(AC1)+\mathbb{P}(AB1 \cap BC1)-\mathbb{P}(AC1 \cap AB1 \cap BC1)$
$\mathbb{P}(A\rightarrow C)=p_{AC}+(p_{AB}\cdot p_{BC})-(p_{AC}\cdot p_{AB}\cdot p_{BC})$
- **(2)** Dal bivacco $A$ si può arrivare in $B$ solamente usando il sentiero $AB$ siccome ci hanno avvisati che non si può andare in $C$ in alcun modo: 
$$\mathbb{P}(A\rightarrow B|(A\rightarrow C)^c)=\frac{\mathbb{P}(A\rightarrow B \cap (A\rightarrow C)^c)}{\mathbb{P}(A\rightarrow C)^c}=\frac{p_{AB}\cdot (1-p_{BC})\cdot (1-p_{AC})}{1-\mathbb{P}(A\rightarrow C)}$$

- **(3)** Indichiamo con $AB_1,AB_2,AB_3$ i tre sentieri che collegano $A$ a $B$ ognuno con prob $q$ di essere percorribile:
    - **(1 bis)** Adesso i nuovi percorsi mi consentono di arrivare a $B$ in altri 2 modi, quindi $\mathbb{P}(A \rightarrow B)=3q-3q^2+q^3$:
    $\mathbb{P}_3(A\rightarrow C)=p\_{AC}+((3q-3q^2+q^3)\cdot p\_{BC})-(p\_{AC}\cdot (3q-3q^2+q^3)\cdot p\_{BC})$
    - **(2 bis)** Risolto **(1 bis)** analogamente al **(2)**:
    $$\mathbb{P}(A \rightarrow B|(A\rightarrow C)^c)=\frac{(3q-3q^2+q^3)\cdot (1-p_{BC})\cdot (1-p_{AC})}{1-\mathbb{P}_3(A\rightarrow C)}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/c0abc004-95e6-48f2-9a11-86e7d8a84672)
